runtime/autoload/provider/clipboard.vim

```diff
@@ -52,6 +52,11 @@ elseif executable('lemonade')
   let s:paste['+'] = 'lemonade paste'
   let s:copy['*'] = 'lemonade copy'
   let s:paste['*'] = 'lemonade paste'
+elseif executable('doitclient')
```

Comment:
We should probably cache the conclusion from this `if..elseif..elseif` chain instead of doing it every time.